scripts/shared/lib/ocp_utils

```diff
@@ -1,5 +1,19 @@
+declare -gr OCP_INSTALLER="${HOME}/.local/bin/openshift-install"
+declare -gA k8s_ocp_version
+k8s_ocp_version[1.20]=latest-4.7
+k8s_ocp_version[1.21]=latest-4.8
+k8s_ocp_version[1.22]=latest-4.9
+k8s_ocp_version[1.23]=latest-4.10
+k8s_ocp_version[1.24]=candidate-4.11
```

Comment:
If we wait a little we’ll be able to use `latest-4.11` (the 4.11.0 installer exists, it just doesn’t have a `latest` link yet).

Reply:
It doesn't matter so much, any `candidate-X.Y` keeps getting updated to the RC anyhow so we can still use it until we update to `latest-4.11`.
The only thing is `candidate-X.Y` is not necessarily GA, but other than that we could theoretically use if for any version..

Reply:
That’s what I thought as well, but the current candidate 4.11 is rc7, not the final 4.11.0 which is available separately (compare https://mirror.openshift.com/pub/openshift-v4/clients/ocp/candidate-4.11/ and https://mirror.openshift.com/pub/openshift-v4/clients/ocp/4.11.0/).

Reply:
All this is moot: https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest-4.11/

Reply:
It had value when it had value :wink: